astropy/io/votable/tests/table_test.py

```diff
@@ -177,3 +177,16 @@ def test_empty_table():
         pedantic=False)
     table = votable.get_first_table()
     astropy_table = table.to_table()
+
+
+def test_validate_xmllint_true(monkeypatch):
+    from ..table import validate
+    from .. import xmlutil
+
+    file = get_pkg_data_filename('data/empty_table.xml')
+
+    def mock_xmllint_validate(filename, version):
+        return 0, 'ok', 'ko'
+    monkeypatch.setattr(xmlutil, 'validate_schema', mock_xmllint_validate)
```

Comment:
Hmm. I wonder if it is possible to only mock the actual `xmllint` subprocess. I'll investigate.

Reply:
I think you'd have to be able to import it to do so.  The logic in the xmlutil.validate_schema is already tested in vo_test with a try except that's why I mocked it direclty there.

Reply:
Ah, ops. I saw this too late! :sweat_smile:

Reply:
No worries :) whatever fits best the astopy way to do things ^^ is the way to go and you should know that better than me ;)